src/modules/print/DailyPrintView.js

```diff
@@ -742,7 +749,10 @@ class DailyPrintView {
       align: 'right',
       width: this.chartArea.leftEdge - this.summaryArea.rightEdge - 3,
     };
-    _.each(this.bgBounds, (bound) => {
+
+    const renderedBounds = _.filter(this.bgBounds, bound => bound <= this.bgScaleYLimit);
```

Comment:
The function notation here gave me pause. `=> var <=` looked pretty strange at a glance.

Reply:
I added some parentheses around the returned comparison.